test/profiles.js

```diff
@@ -2,40 +2,53 @@ var Mixpanel    = require('../lib/mixpanel-node'),
     Sinon       = require('sinon');
 
 // shared test case
-var test_send_request_args = function(test, func, options) {
-    var expected_data = {
-            $token: this.token,
-            $distinct_id: this.distinct_id,
-        };
-    for (var k in options.expected) {
-        expected_data[k] = options.expected[k];
+test_send_request_args = function(test, func, {args, expected, use_modifiers, use_callback} = {}) {
+    var endpoint;
+    var arguments;
```

Comment:
by making a var `arguments` you're shadowing a reserved word that normally references the current function's arguments, so you'll want to choose a different identifier.